src/main/java/duke/parsers/DateTimeParser.java

```diff
@@ -0,0 +1,52 @@
+package duke.parsers;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.TemporalAccessor;
+
+import duke.exceptions.DukeException;
+
+/**
+ * A wrapper around multiple <code>DateTimeFormatter</code>s that parses
+ * user-entered dates/times
+ */
+public class DateTimeParser {
+    private List<DateTimeFormatter> dtFormatters = new ArrayList<>();
+
+    public DateTimeParser() {
+        dtFormatters.add(DateTimeFormatter.ofPattern("dd/MM/yyyy[ HH][:][mm][:ss]"));
+        dtFormatters.add(DateTimeFormatter.ofPattern("dd-MM-yyyy[ HH][:][mm][:ss]"));
+        dtFormatters.add(DateTimeFormatter.ofPattern("dd MM yyyy[ HH][:][mm][:ss]"));
```

Comment:
I like that you have different formatters for DateTime 👍

Reply:
I agree with the above ☝️ 🙂